cmd/checkpoint/main.go

```diff
@@ -86,6 +91,8 @@ func init() {
 	flag.Set("logtostderr", "true")
 	flag.StringVar(&remoteRuntimeEndpoint, "container-runtime-endpoint", defaultRuntimeEndpoint, "[Experimental] The endpoint of remote runtime service. Currently unix socket is supported on Linux, and tcp is supported on windows.  Examples:'unix:///var/run/dockershim.sock', 'tcp://localhost:3735'")
 	flag.DurationVar(&runtimeRequestTimeout, "runtime-request-timeout", defaultRuntimeRequestTimeout, "Timeout of all runtime requests except long running request - pull, logs, exec and attach. When timeout exceeded, kubelet will cancel the request, throw out an error and retry later.")
+	flag.DurationVar(&frequency, "frequency", defaultFrequency, "Rate at which the kubelet and CRI shim is polled for running pods information")
+	flag.DurationVar(&checkPointTimeout, "api-poll-timeout", defaultCheckpointTimeout, "Rate at which the API server is polled for changes to secrets and configmaps")
```

Comment:
Are the defaults reasonable enough to not add flags?

Reply:
Since that's subjective I'm not sure that's a good metric. If we would never want someone to be able to modify them, then we maybe should consider removing them as a flag

Reply:
I'd say let's drop them unless it becomes clear that users need to be able to modify these

Reply:
Done